app/models/redirect.rb

```diff
@@ -3,4 +3,13 @@ class Redirect < ActiveRecord::Base
 
   validates :old_path, presence: true
   validates :new_path, presence: true
+
+  HUMANIZED_ATTRIBUTES = {
+    old_path: 'Redirect source',
+    new_path: 'Redirect destination'
+  }.freeze
+
+  def self.human_attribute_name(attr, options = {})
+    HUMANIZED_ATTRIBUTES[attr.to_sym] || super
+  end
```

Comment:
Have a gander at http://guides.rubyonrails.org/i18n.html#translations-for-active-record-models for an alternate way to do this.